File: Iniciante/PHP/exercise_1036.php
<?php

$valores = readline("Insira 3 valores: ");

list($valorA, $valorB, $valorC) = sscanf($valores, "%f %f %f");

$delta = pow($valorB, 2)-(4*$valorA*$valorC);

if ($valorA > 0 && $delta >= 0) {
    $raiz1 = number_format(((-1*$valorB+sqrt($delta))/(2*$valorA)), 5);
    $raiz2 = number_format(((-1*$valorB-sqrt($delta))/(2*$valorA)), 5);

    print_r("R1 = ".$raiz1."\n");
    print_r("R2 = ".$raiz2."\n");
} else {
    print_r("Impossivel calcular\n");
}

//Concluído!

File: Iniciante/Python/exercises from 1000 to 1099/exercise_1036.py
valorA, valorB, valorC = [float(x) for x in input().split()]

delta = valorB**2 - (4*valorA*valorC)

if valorA > 0 and delta >= 0:
    raiz1 = (-valorB + (delta)**(1/2))/(2*valorA)
    raiz2 = (-valorB - (delta)**(1/2))/(2*valorA)

    print(''.join(['R1 = ', "{:.5f}".format(raiz1)]))
    print(''.join(['R2 = ', "{:.5f}".format(raiz2)]))
else:
    print('Impossivel calcular')